pkg/api/handlers/libpod/networks.go

```diff
@@ -128,7 +128,11 @@ func InspectNetwork(w http.ResponseWriter, r *http.Request) {
 		utils.InternalServerError(w, err)
 		return
 	}
-	utils.WriteResponse(w, http.StatusOK, reports)
+	if len(reports) != 1 {
```

Comment:
I am 95% sure this if is not needed, as we only ask for one name only. Please evaluate this, in the end I can remove it if you want.

Reply:
After looking at this if and functions around for a while, I am now 99% certain that change reports to reports[0] is enough, but I will wait for your feedback.

Reply:
Agreed, `reports[0]` is good here.